docs/diagrams/UiSequenceDiagram.puml

```diff
@@ -1,4 +1,6 @@
 @startuml
+!include docs/diagrams/Style.puml
+
 actor User
```

Comment:
I think there should be some loop in the sequence diagram since we are constantly asking for input but the diagram looks good.